freebasics/templates/base.html

```diff
@@ -1,124 +1,122 @@
 
 {% load wagtailcore_tags compress static wagtailuserbar core_tags freebasics_tags i18n %}
+{% load static from staticfiles %}
 
 <!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8" />
-        <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}{% block title_suffix %}{% endblock %}</title>
```

Comment:
While we're here, can you also add these [social media tags](https://github.com/praekelt/molo-tuneme/blob/develop/tuneme/templates/_base.html#L16-L25)